simpleclient_guava/src/main/java/io/prometheus/client/cache/guava/CacheMetricsCollector.java

```diff
@@ -0,0 +1,79 @@
+package io.prometheus.client.cache.guava;
```

Comment:
EventBus is one that stands out, and I can imagine utility classes around the others.